config.mk

```diff
@@ -1,12 +1,31 @@
-ifneq ($(GAPPS_VARIANT),)
-
 # Opengapps AOSP build system
-GAPPS_BUILD_SYSTEM_PATH := vendor/google/build/core
+GAPPS_BUILD_SYSTEM_PATH := vendor/opengapps/build/core
```

Comment:
Unfortunately, `my-dir` isn't defined when `config.mk` is processed (I think `my-dir` is intended more for use inside `Android.mk` files).  However, those variable definitions could be cleaned up.

I thought about structuring the changes here so that they don't break existing builds, but I'm not certain if that is desirable.  My thinking was that we want people to transition away from using the path `vendor/google/build`.

If my changes are pulled in, but the project is kept under the path `vendor/google/build`, then the build will fail immediately with a message that asks for the path be updated in the manifest.